mavgen/mavgen.go

```diff
@@ -386,7 +386,7 @@ const ({{range .Entries}}
 		e.Name = UpperCamelCase(e.Name)
 		for i, ee := range e.Entries {
 			if ee.Value == 0 {
-				ee.Value = uint8(i)
+				ee.Value = uint32(i)
```

Comment:
It _is_ necessary, because `i` is an index and, consequently, it is of type `int`, not `uint32`